contracts/Receipt.sol

```diff
@@ -37,6 +37,13 @@ contract Receipt is IReceipt, Ownable {
         transferOwnership(gov);
     }
 
+    /**
+     * @param newReleaseTime new release time
+     */
+    function setReleaseTime(uint256 newReleaseTime) external onlyOwner {
+        releaseTime = newReleaseTime;
+    }
+
```

Comment:
Setters should be towards the bottom. E.g. check `setMaxSlippage` in this file.

Reply:
Consult 797bb79.